pkg/redshift/redshift.go

```diff
@@ -314,7 +314,7 @@ func getDistColumnSQL(columns []ColInfo) (string, error) {
 		}
 	}
 	if len(k) == 0 {
-		return "", nil
+		return "diststyle even", nil
```

Comment:
On second thoughts, this may make querying slower for new table refreshes. Need to set `diststyle even` only for staging tables and not for the final tables.

Reply:
@justjkk check this https://github.com/practo/tipoca-stream/pull/262/commits/1e25077a66944938872c80fc606d2e57ba731716

Reply:
👍